build.gradle

```diff
@@ -45,51 +45,7 @@ allprojects {
         }
 
         maven {
-            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
-            credentials {
-                username System.getenv("packageUser")
-                password System.getenv("packagePAT")
-            }
-        }
-
-        maven {
```

Comment:
Any reason to remove maven url?

Reply:
I have deleted it accidentally as this was in middle. Re-added now